src/index.html

```diff
@@ -8,6 +8,101 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <ul class="stars stars--0">
+      <li class="stars__star">
+      </li>
+
+      <li class="stars__star">
+      </li>
+
+      <li class="stars__star">
+      </li>
+
+      <li class="stars__star">
+      </li>
+
+      <li class="stars__star">
+      </li>
+    </ul>
```

Comment:
```suggestion
    <ul class="stars stars--0">
      <li class="stars__star"></li>
      <li class="stars__star"></li>
      <li class="stars__star"></li>
      <li class="stars__star"></li>
      <li class="stars__star"></li>
    </ul>
```
hard ot read, fix it everywhere

Reply:
![7](https://user-images.githubusercontent.com/114413825/198270125-f63a80ca-f6d5-43e3-8705-c02001115b7c.jpg)
In the description to this task we need to make this empty lines. That's why it looks like this.
https://github.com/mate-academy/layout_stars/blob/master/checklist.md

Reply:
these blocks should be singleline, so there is no need in empty lines
![image](https://user-images.githubusercontent.com/99331793/198277011-a2f324ad-6a81-4d5e-8546-f6d0595c161a.png)